src/main/java/com/bugsnag/Bugsnag.java

```diff
@@ -42,6 +42,44 @@ public void rejectedExecution(Runnable runnable, ThreadPoolExecutor executor) {
      * Initialize a Bugsnag client and automatically send uncaught exceptions.
      *
      * @param apiKey your Bugsnag API key from your Bugsnag dashboard
+     * @return The Bugsnag client instance
+     */
+    public static Bugsnag createBugsnag(String apiKey) {
+        return createBugsnag(apiKey, true);
+    }
+
+    /**
+     * Initialize a Bugsnag client.
+     *
+     * @param apiKey                 your Bugsnag API key
+     * @param sendUncaughtExceptions should we send uncaught exceptions to Bugsnag
+     * @return The Bugsnag client instance
+     */
+    public static Bugsnag createBugsnag(String apiKey, boolean sendUncaughtExceptions) {
+
+        // Look for an existing instance of Bugsnag in the appender
+        if (BugsnagAppender.getInstance() != null
+                && BugsnagAppender.getInstance().getBugsnag() != null) {
+
+            // Check that the API key matches
+            if (apiKey.equals(BugsnagAppender.getInstance().getBugsnag().config.apiKey)) {
+
+                // Ensure that sendUncaughtExceptions is respected
+                BugsnagAppender.getInstance().getBugsnag()
+                        .config.setAutoCaptureSessions(sendUncaughtExceptions);
+
+                return BugsnagAppender.getInstance().getBugsnag();
+            }
+        }
+
+        return new Bugsnag(apiKey, sendUncaughtExceptions);
```

Comment:
Some customers initialise Bugsnag multiple times in different parts of their app. I think it would be useful to have some way of disabling the appender on a per-client basis, and generally assuming that the appender will have to co-operate with multiple clients. The `ExceptionHandler` is a good example for how this is handled.

Reply:
I think the most sensible way of handling this would be to have multiple log appenders defined in the XML file.

Added code to handle muliple instances of the log appender defined